Sources/StytchCore/StytchClient/User/StytchClient+User.swift

```diff
@@ -0,0 +1,63 @@
+public extension StytchClient {
+    /// The SDK allows you to manage the current user's information, such as fetching the user, viewing the most recent cached version of the user, or deleting existing authentication factors associated with this user.
+    struct UserManagement {
+        let router: NetworkingRouter<UsersRoute>
+
+        /// Returns the most-recent cached copy of the user object, if it has already been fetched via another method, else nil.
+        public var syncUser: User? {
+            Current.localStorage.user
+        }
+
+        // sourcery: AsyncVariants
+        /// Fetches the most up-to-date version of the current user.
+        public func get() async throws -> UserResponse {
+            try await updatingCachedUser {
+                try await router.get(route: .index)
+            }
+        }
+
+        // sourcery: AsyncVariants
+        /// Deletes, by id, an existing authentication factor associated with the current user.
+        public func deleteFactor(_ factor: AuthenticationFactor) async throws -> UserResponse {
+            try await updatingCachedUser {
+                switch factor {
+                case let .biometricRegistration(id):
+                    return try await router.delete(route: .factors(.biometricRegistrations(id: id)))
+                case let .cryptoWallet(id):
```

Comment:
Are crytpowallet and webauthn already supported in iOS? I'm noticing we're missing these delete methods in Android.

Reply:
The interaction and use of these factors are not yet supported on iOS (there isn't currently a plan to add webauthn functionality at all), but they factors are present in the user object, so I decided to allow deleting existing factors from the user management interface.

Reply:
Alright, I'll make a linear ticket to add these missing routes to Android, too